tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -127,3 +127,35 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+  @env-config
+  Scenario Outline: upload a file with virus with various infected file handling setting set to continue
```

Comment:
```suggestion
  Scenario Outline: upload a virus infected file along with various infected file handling setting set to continue
```
if this seems correct to you.

Reply:
```suggestion
  Scenario Outline: upload a file with the virus with config ANTIVIRUS_INFECTED_FILE_HANDLING set to continue
```
IMO, the scenario is related to setting the config to continue.